docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+.. inclusion-marker-start-do-not-remove
+
+Building a Conda environment with GPU support for Horovod
+=========================================================
+
+In this section we describe how to build Conda environments for my deep learning projects using 
+Horovod to enable distributed training across multiple GPUs (either on the same node or 
+spread across multuple nodes).
+
+Installing the NVIDIA CUDA Toolkit
+----------------------------------
+
+First thing you need to do is to install the `appropriate version`_ of the NVIDIA CUDA Toolkit on 
+your workstation. In this section we are using `NVIDIA CUDA Toolkit 10.1`_ (`documentation`_) 
```

Comment:
Remove "In this section we are using"

Reply:
Removed and cleaned this section up a bit. At time of writing I think CUDA Toolkit 10.1 is the only version of CUDA that can be used with Horovod that doesn't require building one of either PyTorch, TensorFlow, or MXNet from source. Newer version of PyTorch and MXNet support CUDA 10.2 but don't think TF is working properly with 10.2 out of the box.

Reply:
Done